Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM centos:7
+LABEL maintainer="Devtools <devtools@redhat.com>"
+LABEL maintainer="Devtools-test <devtools-test@redhat.com>"
+
+RUN yum install -y git mercurial make binutils bison gcc glibc-devel which
```

Comment:
I doubt we need `mercurial` here. WIT removed it lately.

Reply:
ok, I just copy-pasted it from the setup

Reply:
minimized the stack of packages as part of 3817dbe